proc_scripts/plotting.py

```diff
@@ -28,6 +28,10 @@ def draw_limits(thisrange,s):
             axvspan(j[0],j[1],color='w',alpha=0.5,linewidth=0)
     gca().set_xlim(my_xlim)
 class fl_mod(figlist_var):
+    """
+    .. todo::
+        Alex write a docstring here
```

Comment:
did the best I could. Am not used to doing so please give feedback on what I am missing or better ways to explain as I know my wording is not the best (but I want to get better!)

Reply:
this looked decent to me -- I'm going to add a line with the short description at the beginning, which was missing the first line that gives a 1 sentence summary is actually a required part of the format (it's used if you generate a summary table etc)